src/scripts/main.js

```diff
@@ -1,7 +1,23 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const list = document.body;
```

Comment:
'list' is not an appropriate name for this element. You should use 'body' instead